R/jira.R

```diff
@@ -8,15 +8,16 @@
 
 #' Parse Jira issue and comments
 #'
+#' Note the comments element will be empty if the downloaded json only contain issues.
```

Comment:
The description of the function should not read like an observation. May want to iterate with the group on this a bit or see other functions docs. try checking the docs of other libraries like data.table.